components/Header.tsx

```diff
@@ -0,0 +1,59 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import React from 'react'
+import GithubSvg from '../assets/github_thumbnail.svg'
+import DiscordSvg from '../assets/discord_thumbnail.svg'
+import { DiscordLink, GithubLink } from '../constants/urls'
+
+const LinkSets = [
+	{ name: '關於SPT', url: '#' },
+	{ name: '聯絡我們', url: '#' },
+	{ name: '技術支援', url: '#' }
+]
```

Comment:
I think renaming it to 'links' is better.

It's an array instead of a Set in JavaScript. 

Because JavaScript actually has the Set data type, which may cause confusion.

Reply:
Ok, i'll change it from `LinkSets` to `LinkConfig`